atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -2272,10 +2274,11 @@ public void cleanUpSchemaMutationLockTablesState() throws Exception {
         tables.remove(tableToKeep.get());
         if (tables.size() > 0) {
             dropTablesInternal(tables);
-            log.info("Dropped tables [{}]", tables.toString());
+            log.info("Dropped tables [{}]", UnsafeArg.of("table names", tables.toString()));
```

Comment:
Do we really need to wrap unsafe args? There's a non-zero allocation & GC cost to wrapping for logging in tight loops or on hot code paths (though this specific case is not one).

Also realize this was like this before, but we should not be calling `toString()` on any log args as that kind of defeats the point of lazy evaluation if we're not logging at that log level.